cmd/provider.go

```diff
@@ -28,6 +28,7 @@ func makeProviderCmd() *cobra.Command {
 	}
 
 	command.Flags().String("pull-policy", "Always", `Set to "Always" to force a pull of images upon deployment, or "IfNotPresent" to try to use a cached image.`)
+	command.Flags().String("nameserver", "8.8.8.8", `Set to the DNS server you want to use for resolving DNS queries`)
```

Comment:
I would expect this to be a string array so that multiple addresses can be given, it seems standard to see 8.8.8.8 and 8.8.8.4 or the equivalent set of Cloudflare IPs.